plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -179,10 +188,23 @@ def get_ceph_versions_mismatch(self):
             for version in versions[daemon_type]:
                 vers.append(version)
                 global_vers.add(version)
-
+                if self.version_compare(str(version)) > \
+                        self.version_compare(h_version):
+                    h_version = version
+                    h_daemon = daemon_type
             if vers:
                 daemon_version_info[daemon_type] = vers
 
+        for daemon_type in versions:
+            if daemon_type == 'mon':
+                for version in versions[daemon_type]:
```

Comment:
You could shorted the above 3 lines to:

for version in versions.get('mon', []):

Reply:
ack thank you